docs/advanced.rst

```diff
@@ -1098,6 +1098,14 @@ pybind11 will automatically and transparently convert
 1. Static and dynamic Eigen dense vectors and matrices to instances of
    ``numpy.ndarray`` (and vice versa).
 
+1. Returned matrix expressions such as blocks (including columns or rows) and
```

Comment:
Oops; I saw the old repeated 1. and assumed it worked like Markdown, but [apparently not](http://pybind11.readthedocs.io/en/latest/advanced.html#transparent-conversion-of-dense-and-sparse-eigen-data-types)